db.py

```diff
@@ -124,4 +128,47 @@ def savequiz(data, user):
     else:
         db.profile.insert({"userId": user, "quizAnswers": answers})
         return answers
-      
\ No newline at end of file
+        
+        
+def loadGroupMessages(groupId):
+    message = { "sender" : "", "timestamp" : "", "message" : ""}
+    messages= []
+    results= []
+    try:
+        _id= ObjectId(groupId)
+        query= {"_id" : _id }
+        group= db.groupchat.find_one(query)
+        groupMessages= group['messages'][0]
+        for m in groupMessages:
+            results.append(db.messages.find_one(ObjectId(m)))
+            print(m)
+            
+        for result in results:
+            sender= db.users.find_one(result["sender"])
+            if sender is not None:
+                message["sender"] = sender["username"]
+            else:
+                sender= db.googleUsers.find_one(result["sender"])
+                message["sender"] = sender["email"]
+            message["timestamp"]= result["createTimestamp"]
+            message["message"] = result["message"]
+            print(message)
+            messages.append(message)
+        return messages
+   
+    except Exception as e:
+        print(e)
+        print("No messages found")
+    
+def userChats(username):
+    returnedGroups= []
+    userchats= db.groupchat.find({"users.id": username})
+    for result in userchats:
+        group= Group(result["_id"], result["name"], result["users"], result["createTimestamp"], result["description"],
+            result["photo"], result["messages"])
+        returnedGroups.append(json.dumps(group.__dict__))
+    print(returnedGroups)
+    return returnedGroups
+         
+    
+
```

Comment:
This code appears to be a part of another PR, just as a heads up. However, it looks great in terms of allowing the user's chats to come through within the groupchat. It seems to be a great addition to the project and should provide a better user experience.
<img src="https://media3.giphy.com/media/wS6QbYsNxxgpdt9bsV/giphy.gif"/>